app/src/main/java/com/rob729/quiethours/Fragments/DetailsFragment.kt

```diff
@@ -36,6 +35,8 @@ class DetailsFragment : BottomSheetDialogFragment() {
         }
     }
     private var days: List<Boolean> = ArrayList()
+    private var _binding: FragmentDetailsBinding? = null
+    private val binding get() = _binding!!
```

Comment:
get() has to be put in the next line.

Reply:
next line means, below the @SuppressLint("SetTextI18n") but outside onCreateView, right?

Reply:
it should be 
private val binding
    get() = _binding!!

Reply:
I have done this, I will push the code